packages/grid/rathole/rathole.dockerfile

```diff
@@ -0,0 +1,26 @@
+ARG RATHOLE_VERSION="0.5.0"
+ARG PYTHON_VERSION="3.12"
+
+FROM rust as build
+ARG RATHOLE_VERSION
+ARG FEATURES
+RUN apt update && apt install -y git
+RUN git clone -b v${RATHOLE_VERSION} https://github.com/rapiz1/rathole
+
+WORKDIR /rathole
```

Comment:
Reverted to compling rathole, as it current image on dockerhub doesn't have binaries for arm